src/main/java/org/passay/WhitespaceRule.java

```diff
@@ -100,15 +181,33 @@ public RuleResult validate(final PasswordData passwordData)
   }
 
 
+  /**
+   * Creates the rule result metadata.
+   *
+   * @param  password  data used for metadata creation
+   *
+   * @return  rule result metadata
+   */
+  protected RuleResultMetadata createRuleResultMetadata(final PasswordData password)
+  {
+    final Map<String, Object> m = new LinkedHashMap<>();
+    m.put(
+      "whitespaceCharacterCount",
```

Comment:
I think these keys are overly wordy. _whitespaceCount_ seems perfectly clear and more succinct besides.

Reply:
I can change that.